modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/utils/handleMultiSelect.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ACCOUNTS_DND_ENABLED} from './flags';
+
+let items = null;
```

Comment:
Mhh... These utility functions are not meant to be used by a react component and `items` is something the caller should not be aware of. 

Anyway I can make this cleaner (or at least more understandable) by transforming this file in a js class and `items` in a class `attribute`. Does it sound ok?

Reply:
I've updated the PR, @markocikos please check if it's ok for you now :)

Reply:
@FabioDiegoMastrorilli  Yeah, a class would make more sense.  The key thing to have in mind is that this is **not** a utility function. A utility function does not have a state, if does not depend on any value set by an another function. People using it should feel confident that it will work consistently at any time, from any context. 

It it has a state, it is a component. People need to make an instance of it, to be able to control or track its state from parent context. If there are two of these on a page, each component has its own state. If you had the code as it is, it would break if you ever had two usages of it on a page. A class fixes this, as you can have instances.

Although functions, I should note that this is pretty unusual design for a web UI, to have a component without UI. I don't know the broader context, maybe there is a need for it. But I would definitely step back here and take a look at the broader picture, to see if code can be organized in a more cleaner way.

Reply:
It totally makes sense :) Thanks for the explanation! We can eventually have a look together after the fourth PR (which will only contain some refinements and little improvements will be merged)